### PR TITLE
SONAR-7319 WS api/ce/task_types list CE task types

### PR DESCRIPTION
**Pull request by teryk**

### CODE REVIEW

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'protobuf' package (org.sonarsource.sonarqube:sonar-db:target/generated-sources/protobuf/org/sonar/db/protobuf) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'protobuf' package (org.sonarsource.sonarqube:sonar-db:target/generated-sources/protobuf/org/sonar/db/protobuf) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)

File: server/sonar-server/src/main/java/org/sonar/server/computation/ws/TaskTypesAction.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    for (CeTaskProcessor taskProcessor : taskProcessors) {
+      taskTypesBuilder.addAll(taskProcessor.getHandledCeTaskTypes());
+    }
+    this.taskTypes = taskTypesBuilder.build();
+  }
+
+  @Override
+  public void define(WebService.NewController controller) {
+    controller.createAction("task_types")
+      .setDescription("List task types available")
```
Comment (simonbrandhof) [inline server/sonar-server/src/main/java/org/sonar/server/computation/ws/TaskTypesAction.java]: Isn't "List available types of tasks" better ?

**Comment by simonbrandhof** [comment]
LGTM

**Pull request closed without merging**